### ISSUE TITLE
Handler: Select to avoid drag&drop 

### ISSUE THREAD

**Issue by ghigio2000** [Studio-42/elFinder#2630]
I need to do "something" once a file is clicked...but I need to avoid it when the file is selected for drag and drop, or right mouse clicked.
I am using the "handlers: select" but it is fired also when drag&drop and mouse right click on the file.
Is there a solution? 
P.S.: on "dblclick" I like the file download.

**Comment by nao-pon** [comment]
@ghigio2000 One solution is to monitor click events.

```javascript
var elf = $('#elfinder').elfinder('instance');
elf.getUI('cwd').on('click', '[id]', function(e) {
    var file = elf.file($(this).attr('id'));
    elf.log(file);
});
```

**Comment by ghigio2000** [comment]
I thought it was not possible as I did not see listed the "click" in your handerls events.
I will give it a try, thanks.

**Comment by ghigio2000** [comment]
Meanwhile, I am not very expert programmer as you might see, I would like to debug some "rename.pre" I have added to the connector, but I am not able to find a way while using the Firefox "console web" on my html page. 
I tried sending from the php some "echo", "print_r", "var_dump" but I do not see them on the console. I also enabled the "E_ALL" on the error_reporting, but I suspect the console web is not the right instrument.
Any suggestion is very welcome?
